entries/a/australianretirementtrust.com.au.json

```
@@ -5,16 +5,15 @@
       "sunsuper.com.au",
       "qsuper.qld.gov.au"
     ],
-    "contact": {
-      "facebook": "AustralianRetirementTrust",
-      "twitter": "Ausretiretrust",
-      "form": "https://www.australianretirementtrust.com.au/contact-us/email-us"
-    },
+    "tfa": [
+      "sms",
+      "email"
+    ],
```

```suggestion
    ],
    "notes": "Requires contacting customer service to activate",
```

There should be a message like this. If you have a better way of phrasing it, use that instead.